lib/spark.js

```diff
@@ -16,7 +16,12 @@ export default class Spark {
   async getRetrieval () {
     console.log('Getting retrieval...')
     const res = await this.#fetch('https://spark.fly.dev/retrievals', {
-      method: 'POST'
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        sparkVersion: SPARK_VERSION,
+        zinniaVersion: Zinnia.versions.zinnia
```

Comment:
Do we want to handle the case when running on an older Zinnia version?

```suggestion
        zinniaVersion: Zinnia.versions?.zinnia
```

Reply:
I don't think so, let's just bump the required zinnia version with this one. We control when Spark gets updated, and can upgrade Zinnia in the same commit